src/components/Recidiviz/RecidivizForm/RecidivizForm.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import RecidivizForm from './RecidivizForm';
+
+jest.mock('../../../lib/static/nlform.js');
+jest.mock('../../../lib/static/likelihoodSelect.js');
+
+it('renders the RecidivizForm component', () => {
```

Comment:
More test possibilities: checking a variety of combinations and asserting the proper estimate, including the edge case where the combination is unknown and we default to the max time.

Reply:
Given the stance of treating all of the legacy code that you wrote (or modified) as third-party, I don't believe we should add these types of tests _in this PR_. I would really like to rewrite all of that code into the existing React components, or create additional components, and add tests at that point in time.